evap/evaluation/migrations/0132_add_infotext_model.py

```diff
@@ -1,73 +1,4 @@
-<em>Mehr Infos: <a href="/faq#faq-3-s">FAQ/Ergebnisse</a></em>""",
-            Infotext.Page.CONTRIBUTOR_INDEX,
-        ),
-        (
-            "Information about the evaluation",
-            "Informationen zur Evaluierung",
-            """<b>Anonymity</b><br>
-Your votes and text answers can't be related to you. But you should be aware that your style of writing might allow lecturers to guess who wrote the text answer, especially in small courses.<br>
-<em>More details: <a href="/faq#faq-21-q">FAQ/Anonymity</a></em><br>
-<br>
-<b>References to other answers</b><br>
-Lecturers can't see completed questionnaires as a whole. If you would write "see above", the lecturer can't find the respective answer.<br>
-<em>More details: <a href="/faq#faq-24-q">FAQ/Reference</a></em><br>
-<br>
-<b>Evaluation Results</b><br>
-Text answers will be shown to the people who were evaluated and to the people responsible for the course. Voting answers will be published for all users of the platform if at least two people participated in the evaluation. The average grade is calculated if the participation rate is at least 20 percent.<br>
-<em>More details: <a href="/faq#faq-3-s">FAQ/Results</a></em>""",
-            """<b>Anonymität</b><br>
-Deine abgegebenen Stimmen und Textantworten können dir technisch nicht zugeordnet werden. Dir sollte aber bewusst sein, dass dich Mitwirkende insbesondere in kleinen Veranstaltungen an deinem Schreibstil erkennen könnten.<br>
-<em>Mehr Infos: <a href="/faq#faq-21-q">FAQ/Anonymität</a></em><br>
-<br>
-<b>Verweise auf andere Antworten</b><br>
-Lehrende können sich keine kompletten Fragebögen ansehen. Wenn du in einer Textantwort schreibst "wie oben schon erwähnt", können sie die zugehörige Antwort nicht finden.<br>
-<em>Mehr Infos: <a href="/faq#faq-24-q">FAQ/Verweise</a></em><br>
-<br>
-<b>Evaluierungs-Ergebnisse</b><br>
-Textantworten werden den bewerteten Personen und den Verantwortlichen der Veranstaltung angezeigt. Antworten auf Abstimmungsfragen werden für alle Nutzer·innen der Plattform veröffentlicht, wenn mindestens zwei Personen an der Evaluierung teilgenommen haben. Durchschnittsnoten werden berechnet, wenn die Teilnahmequote mindestens 20 Prozent beträgt.<br>
-<em>Mehr Infos: <a href="/faq#faq-3-s">FAQ/Ergebnisse</a></em>""",
-            Infotext.Page.STUDENT_INDEX,
-        ),
-        ("", "", "", "", Infotext.Page.GRADES_PAGES),
-    ]
-
-    for title_en, title_de, content_en, content_de, page in infotexts:
-        infotext.objects.create(title_en=title_en, title_de=title_de, content_en=content_en, content_de=content_de,
-                                page=page)
```

Comment:
I think @janno42's comment aimed at removing the actual infotext contents and titles from the migration, but we'd still need to create the three (empty) objects in the database, as our code relies on them existing.

Reply:
All right. Can I assert in my tests, that all infotexts are empty per default?

Reply:
Yes, if the intention is to have a "before, they are empty, and after page submission, they have the expected value" test. Otherwise, the assertion probably has limited use :D